### ISSUE TITLE
Pieces Copilot shows nothing

### ISSUE THREAD

**Issue by gigit0000** [pieces-app/support#366]
### Software

VS Code

### Operating System / Platform

Windows

### Your Pieces OS Version

Latest

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

Hi, I installed the package, but in VSCode, it doesn't show anything as in the screenshot. Maybe it's not figured correctly. Can anyone
![ddd](https://github.com/user-attachments/assets/dbe6cb2b-dfef-464c-9aa7-56b459f13695)
 help?

**Comment by gigit0000** [comment]
I did, but it didn't work. I think it's related to Pieces OS, which doesn't open at all.

**Comment by mack-at-pieces** [comment]
Hey @gigit0000 - Can you access http://localhost:1000/.well-known/version in your browser and share a screenshot here? Thanks!

**Comment by gigit0000** [comment]
Hi, It's 10.1.1

**Comment by mack-at-pieces** [comment]
Okay that's perfect. That was to make sure that Pieces OS is successfully running. Do you happen to have any security software or firewalls running that might interfere with your ability to access a static site on a localhost port?

**Comment by gigit0000** [comment]
Oh, yes. But when I installed Pieces, I allowed it as an exception in Avast.

**Comment by mack-at-pieces** [comment]
It's possible that may still be interfering. Would you be willing to turn off any blockers and give it a try to see if that's the cause?

**Comment by gigit0000** [comment]
Unfortunately, it still doesn't work. 

BTW, I just found Pieces OS in the hidden block of task bar. Should it always be running on Windows?
![rrr](https://github.com/user-attachments/assets/09f57313-9a0f-4560-9c76-69a5c4a0d0fd)

**Comment by gigit0000** [comment]
Never mind. I'll try out one day again. Thanks for your support.

**Comment by ynhong1** [comment]
I faced the same problem, the Pieces Copilot doesn't show anything in my VS code. But it's wired because the Pieces Desktop is working correctly. It seemed problem is related to VS code integration, not security software or firewalls.
![image](https://github.com/user-attachments/assets/800aa0b3-6a17-41d9-a3ab-009c59654396)
![image](https://github.com/user-attachments/assets/ea735618-eef6-41fb-af50-431c2bdb8ec6)

**Comment by ynhong1** [comment]
After I forcely shutdown the Pieces OS, and then re-start Visual Studio Code again, my problem seemed disappeared. The Pieces Copilot VS code plug-in can run correctly now.

**Comment by rosie-at-pieces** [comment]
Hi everyone, 

We're trying to track this issue down, would you please use [this link](https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ2iaRWpd1kA2k_xg2KrVi_jiTHo0POiMZSI9WAYsxs5jJ9sU2Rd2zKitLuSFd7l3FhatGzy5Vgt) to set up a call with Mark? And if you're available today but don't see a time please just email mark@pieces.app and he'll get you in!

**Comment by joberror** [comment]
Yes! I am facing this exact issue currently, Pieces Explorer is working fine, but Pieces Copilot isn't showing anything. Restarted VS Code many times, shutdown the PC, but still the same. Everything else is working aside Pieces Copilot not showing anything. 

it started recently.
